src/IBM.Cloud.SDK.Core/Service/IBMService.cs

```diff
@@ -83,15 +83,15 @@ protected IBMService(string serviceName, IAuthenticator authenticator)
 
             // Try to retrieve the service URL from either a credential file, environment, or VCAP_SERVICES.
             Dictionary<string, string> props = CredentialUtils.GetServiceProperties(serviceName);
-            props.TryGetValue(propnameUrl, out string url);
+            props.TryGetValue(propNameServiceUrl, out string url);
             if (!string.IsNullOrEmpty(url))
             {
                 SetEndpoint(url);
             }
 
             // Check to see if "disable ssl" was set in the service properties.
             bool disableSsl = false;
-            props.TryGetValue(propnameDisableSsl, out string tempDisableSsl);
+            props.TryGetValue(propNameServiceDisableSsl, out string tempDisableSsl);

```

Comment:
Should be `propNameDisableSsl`.

Reply:
There are two disable ssl constants - one at the auth level and one at the service level

```
// in Authenticator
public static string PropNameDisableSsl = "AUTH_DISABLE_SSL";
// In base service
public static string propNameServiceDisableSsl= "DISABLE_SSL";
```

I want to differentiate them if possible. What do you think?

Reply:
It makes sense to keep them separate then, one more question I have is that we have `disableSslVerification` instead of `DisableSsl` in unity and ruby and I think a bunch of other SDK's have it that way too. I think `disableSslVerification` is more descriptive.

Reply:
Ok I will at least change the name to UpperCamelCase and update to disableSsslVerification.